lib/Core/Executor.cpp

```diff
@@ -4292,7 +4292,7 @@ void Executor::executeMemoryOperation(
           ObjectState *wos = state.addressSpace.getWriteable(mo, os);
           wos->write(offset, value);
           state.symbolicError->executeStore(address, value, error,
-                                            valueWithError);
+                                            valueWithError, target);
```

Comment:
@Himeshi If the last argument's type is changed to `llvm::Instruction *`, then `target` must change to `target->inst`.